doc/_static/favicon/generate-png.sh

```diff
@@ -0,0 +1,9 @@
+for SIZE in 16 32 96 128 196
+do
+	convert -background none -resize ${SIZE}x${SIZE} icon.svg icon-${SIZE}x${SIZE}.png
```

Comment:
sn't SVG supported as favicon itselft?

Reply:
My experience in web design is close to zero, so I provide both svg and different sizes png (same set as in tarantool.io) just in case. So I don't know if all relevant browsers support svg favicons or not.

![image](https://user-images.githubusercontent.com/20455996/194863300-e9c3c802-1270-427b-94c7-83563d3e3968.png)

Reply:
https://caniuse.com/link-icon-svg